19-strings-runes-bytes/exercises/03-rune-manipulator/main.go

```diff
@@ -18,15 +18,15 @@ package main
 // ---------------------------------------------------------
 
 func main() {
-	strings := []string{
+	words := []string{
```

Comment:
I think it's better to not use the name `strings` as the slice's name because users might use stuff from the `strings` package, and it might create unnecessary confusion.

Reply:
You're right.